engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlan.java

```diff
@@ -307,6 +307,11 @@ public void sendMetrics(Collection<String> metricNames)
       throw new UnsupportedOperationException("Not supported yet.");
     }
 
+    @Override
+    public OperatorMeta getOperatorMeta()
```

Comment:
There is already a getOperatorWrapper method. How about renaming it to getOperatorMeta instead of creating a new method.